modules/contact/test/tests/mortar_aux_kernels/tests

```diff
@@ -90,8 +90,9 @@
     input = 'block-dynamics-aux-fretting-wear-test.i'
     rel_err = 1.0e-3
     abs_zero = 2.0e-4
-    min_parallel = 1
-    max_parallel = 4
+    # debug mesh only runs in serial
+    max_parallel = 1
+    max_threads = 1
```

Comment:
Is there a problem with threads?

Reply:
It crashes with threads, but that test has `debug_mesh = true` so it's not intended to work with threads?

Reply:
Hmm I'm pretty sure you created `debug_mesh` so you'll have to tell me whether it's intended to work with threads haha

Reply:
If it's not supposed to work with threads then we should add an assertion or something to that effect

Reply:
> you'll have to tell me whether it's intended to work with threads haha

That capability uses  `write_equation_systems` in libMesh. It may be able to run in parallel but I have never used it to that effect. I think I was mostly thinking of the mortar statistics --which is handled via the same boolean, but that shouldn't make the test fail.

Reply:
I am reverting that change.